dashboard/app/models/level_source_image.rb

```diff
@@ -25,6 +25,12 @@ def save_to_s3(image)
     return false if CDO.disable_s3_image_uploads
     return false if image.blank?
 
+    begin
+      image = ImageLib::to_png(image)
```

Comment:
Oh, and I noticed that chrome on macosx is ok with jpegs that are named .png. So possibly we could just not change the filename. But I'm not sure if all other browsers are.